starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +18,48 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(array) {
+  var longestword = array[0];
```

Comment:
You shouldn't initiate a variable like this, if the array is empty (which is the case in some test) it will return an error (as there will be no element at index 0).

Instead you can initiate it like this : 

```js
var longestWord = ""

```

Reply:
Also if you want to resolve the error you need to deal with some special cases like an empty array, so you can add another if statement at the beginning of the function, and last point, in your code as you are returning longestword as soon as the condition is verified, it won't work (the function will stop as soon as it reach a word that is longer than the value inside the longest variable and the loop will stop), you need to reassign the value & to return it after the loop like this : 

```js 

 if (array.length == 0) {
    return undefined;
  }
  var longestword = "";
  for (index = 0; index < array.length; index++) {
    if (array[index].length > longestword.length) {
      longestword = array[index];
    } else continue;
  }

```